### ISSUE TITLE
Bug on preview within unified search

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/notes#957]
I have noticed recently that while the Notes search feature still works as far a showing correct preview of what is found, I cannot click on the preview to get to the actual note. This may be restating this same problem, but in any event, if actual url links are provided, it would be one way of solving the problem. This problem occurs using Safari browser on macOS Ventura. Using Chrome browser on same system the linking process to the found notes items works.

_Originally posted by @aardquark in https://github.com/nextcloud/notes/issues/764#issuecomment-1407497549_

**Comment by tobiasKaminsky** [comment]
@aardquark I created a separate issue for your bug repor.

**Comment by juliushaertl** [comment]
@aardquark Can you maybe elaborate a bit more on the report? I don't see an issue with clicking the notes search result here. Maybe you can show a quick screen recording?

**Comment by aardquark** [comment]
Here is a visual of a simple search that I just did while viewing the *Notes* application in the Safari browser on macOS 13.1:
![Screenshot 2023-01-30 at 3 34 21 PM](https://user-images.githubusercontent.com/8345166/215589558-619309b0-29f0-4ad3-a2c8-e7485051c2fc.png)
The currently selected item in the list of found items is denotes by the gray background surround the item that starts with _lake house column repair_.  Now I want see the full text of the note item (i.e. open it in the main portion of the browser window). 

In order to do this I normally just move my mouse to the currently selected item and click. This no longer works on Safari (version is 16.2), but it used to work. Unfortunately, I don't use this feature enough to pinpoint exactly when it stopped working. I can state that I just tested on macOS Catalina and clicking on an item does not work there (in Safari 15.6.1) either. This feature (clicking to open) does work in the Chrome browser.

What does work in either Safari or Chrome is to navigate to the desired item using up/down keys and press enter, which causes the currently selected item in Search to open.

**Comment by juliushaertl** [comment]
Can you check if this is also an issue with the other apps maybe e.g.if you search for files? As far as I see there is no specific handling in the notes app for this.

**Comment by aardquark** [comment]
Yes, I get this same behavior if I am currently using the Files app, that is, the search button (magnifying glass in upper right) finds what I am looking for, but on Safari I must use the Return key because mouse click does not open the item.

Note that with the separate Full Text Search app (the magnifying glass in upper left) mouse click works normally with the found results.

**Comment by juliushaertl** [comment]
Let me move this to server then, since this is likely a general problem with Safari in the unified search.
